test/test_retryable_writes.py

```diff
@@ -490,6 +490,7 @@ def setUpClass(cls):
         }
 
     @client_context.require_version_min(4, 0)
+    @client_knobs(heartbeat_frequency=0.05, min_heartbeat_interval=0.05)
```

Comment:
This was just a slow test I noticed. Decreasing these knobs decreases the runtime from 10 seconds to a few milliseonds.